src/aosm/azext_aosm/deploy/deploy_with_arm.py

```diff
@@ -403,7 +405,8 @@ def deploy_nsd_from_bicep(
             logger.info(message)
             print(message)
             with open(parameters_json_file, "r", encoding="utf-8") as f:
-                parameters = json.loads(f.read())
+                parameters_json = json.loads(f.read())
+                parameters = parameters_json["parameters"]
```

Comment:
This code looks the same in 3 or 4 places.  Can you commonalize it or better read the file once and make parameters a property on the class?